### DISCUSSION TITLE
Hover-card on hover of @-mentions

### DISCUSSION THREAD

**Discussion by ankitsharma-99** [facebook/lexical#4291] [Q&A] [answered]
I want to show a hover-card when hovering over @-mentioned users. Any ideas on how that can be done?

**Comment by ivailop7** [comment, marked as answer]
Hi, you can extend the Mention node. In the createDOM method you can apply a custom CSS class and define that CSS class with ":hover" and render any popover.

**Reply by ankitsharma-99** [reply]
Yep, thanks! That works!
